website/content/en/docs/concepts/settings.md

```diff
@@ -45,10 +45,14 @@ data:
   # faster than this time, the batching window will be extended up to the maxDuration. If they arrive slower, the pods
```

Comment:
Docs are codegenned from the `preview` folder. Can you migrate this change there?

Reply:
Fixed in next revision